trex.org

```diff
@@ -707,6 +728,28 @@ prim_factor =
   #+end_src
   :end:
 
+* Cell (cell group)
+
+  #+NAME: cell
+  | Variable | Type    | Dimensions | Description |
+  |----------+---------+------------+-------------|
+  | ~a~      | ~float~ | ~(3)~      |             |
+  | ~b~      | ~float~ | ~(3)~      |             |
+  | ~c~      | ~float~ | ~(3)~      |             |
+
```

Comment:
I don't like `a`,`b`, and `c` as names.  :-)
Can you find something more explicit? (Maybe ask Sandro)
Also, can you add a description?

Reply:
It was Sandro who proposed this notation :-) 
We can discuss the notations and descriptions during the WP2 meeting.